doc/source/erlangen_hoechstadt_de.md

```diff
@@ -0,0 +1,27 @@
+# Erlangen-Höchstadt
+Support for Landkreis [Erlangen-Höchstadt]() located in Bavaria, Germany.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: erlangen_hoechstadt_de
+      args:
+        city: CITY
+        street: STREET
+  separator: " / "
+```
+Note: Set the `separator` to `" / "` as it's the way different waste types are separated in one calendar event.
```

Comment:
I don't understand this not. `separator `is used to join waste types for the user interface. Therefore it should be irrelevant in the example. Why do you need it here?

Reply:
Because in the ICS Calendar, multiple waste types are displayed e. g. as "Restmülltonne / Biotonne" in one event. So you'll have to split them by configuring it via the `seperator`. Or did I understand it wrong?

Reply:
`separator` is used to join waste types. In your case you have to split waste types. Please check the `split_at` argument of class `ICS`:
https://github.com/mampfes/hacs_waste_collection_schedule/blob/0f6d7b63cb35c63f7725c1d17b7ebae86c12a25c/custom_components/waste_collection_schedule/waste_collection_schedule/service/ICS.py#L16

Reply:
Okay, thanks, I understood and fixed it.